app/views/layouts/_weglot.html.erb

```diff
@@ -0,0 +1,43 @@
+<script type="text/javascript" src="https://cdn.weglot.com/weglot.min.js"></script>
+<script>
+    Weglot.initialize({
+        api_key: 'wg_998f62092305ff602e8efd215c8fd0633',
```

Comment:
does this need to be more private? it's a public repo.

Reply:
(env var)

Reply:
Yeah, I'd move IF we decide to use this tool.